src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -11,52 +10,75 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal static class Resolve
+    internal class DependencyResolver
     {
-        public static (string content, object file) ReadFile(string path, object relativeTo, List<Error> errors, DependencyMapBuilder dependencyMapBuilder, GitCommitProvider gitCommitProvider)
-        {
-            Debug.Assert(relativeTo is Document);
+        public readonly BookmarkValidator BookmarkValidator = new BookmarkValidator();
```

Comment:
Why do we prefer public read-only field to read-only property?

Reply:
both are fine as long as they readonly.